runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/graph/ExecutableStage.java

```diff
@@ -81,53 +81,53 @@
    * follows:
    *
    * <ul>
-   *   <li>The {@link PTransform#getSubtransformsList()} contains no subtransforms. This ensures
+   *   <li>The {@link PTransform#getSubtransformsList()} is empty. This ensures
    *       that executable stages are treated as primitive transforms.
    *   <li>The only {@link PCollection} in the {@link PTransform#getInputsMap()} is the result of
    *       {@link #getInputPCollection()}.
    *   <li>The output {@link PCollection PCollections} in the values of {@link
    *       PTransform#getOutputsMap()} are the {@link PCollectionNode PCollections} returned by
    *       {@link #getOutputPCollections()}.
-   *   <li>The {@link FunctionSpec} contains an {@link ExecutableStagePayload} which has its input
-   *       and output PCollections set to the same values as the outer PTransform itself. It further
-   *       contains the environment set of transforms for this stage.
+   *   <li>The {@link PTransform#getSpec()} contains an {@link ExecutableStagePayload} which has its
+   *       input and output PCollections set to the same values as the outer PTransform itself. It
+   *       further contains the environment set of transforms for this stage.
```

Comment:
"The {@link PTransform#getSpec()} contains an {@link ExecutableStagePayload} with inputs and outputs equal to the PTransform's inputs and outputs, and transforms equal to the result of {@link #getTransforms}"

Reply:
Done